### ISSUE TITLE
Is Co-processor specification accept?

### ISSUE THREAD

**Issue by wyatt-wong** [larsbrinkhoff/awesome-cpus#48]
Is Co-processor specification accept?

**Comment by larsbrinkhoff** [comment]
Yes, I think floating-point, MMU, and similar coprocessors would be appropriate.

**Comment by wyatt-wong** [comment]
> Yes, I think floating-point, MMU, and similar coprocessors would be appropriate.

Thank you. I will try to provide update for co-processors as well.

**Comment by larsbrinkhoff** [comment]
Thanks!
